Gera/Ejercicios UBA/Funciones/Ejercicio11.py

```diff
@@ -0,0 +1,39 @@
+""". Contar la cantidad de letras de un telegrama que termina en punto sin utilizar
+funciones de string, salvo la que indica la longitud"""
+
+
+def ingresar_texto_telegrama():
+    telegrama = input("ingrese el texto del telegrama: ")
+    telegrama = telegrama.lower()
+    return telegrama
+
+
+def convertir_telegrama_en_lista(telegrama):
+    lista_telegrama = list(telegrama)
+    return lista_telegrama
+
+
+def contar_letras_en_telegrama(lista_telegrama):
+    alfabeto = 'abcdefghijklmnñopqrstuvwxyzáéíóú'
+    letras_contadas = 0
+    for i in range(0, len(lista_telegrama)):
```

Comment:
Recordá que los iterables se pueden recorrer por índice o por valor. A vos te interesa el valor en este caso, lo podés simplificar de la siguiente manera:

for letra in telegrama:
     if letra in alfabeto:
         letras_contadas += 1